mne/defaults.py

```diff
@@ -26,7 +26,7 @@
               eeg=(-200., 200.), misc=(-5., 5.),
               seeg=(-200., 200.), dipole=(-100., 100.), gof=(0., 1.)),
     titles=dict(eeg='EEG', grad='Gradiometers', mag='Magnetometers',
-                misc='misc', seeg='sEEG', dipole='Dipole',
+                misc='misc', seeg='sEEG', dipole='Dipole', eog='EOG',
```

Comment:
the `ylims` in the defaults file are ignored by this function. But it does make sense to have an ylim in the defaults for EOG. For example, if you create a custom layout with both EEG and EOG channels (maybe the EOG in a separate column) and use `plot_evoked_topo`, *then* you would need an `ylim` for EOG. At any rate, I cannot think of a reason to *not* have a default ylim for EOG channels.